src/components/AccountContainer.js

```diff
@@ -7,54 +7,37 @@ class AccountContainer extends Component {
   constructor() {
     super()
 
-    // we have provided this default state for you,
-    // use this to get the functionality working
-    // and then replace the default transactions with a call to the API
-
     this.state = {
       searchTerm: '',
-      transactions: [
-        {
-          id: 1,
-          posted_at: "2017-02-28 11:00:00",
-          description: "Leather Pants, Gap co.",
-          category: "Fashion",
-          amount: -20000
-        },
-        {
-          id: 2,
-          posted_at: "2017-02-29 10:30:00",
-          description: "Paycheck from Bob's Burgers",
-          category: "Income",
-          amount: 100000
-        },
-        {
-          id: 3,
-          posted_at: "2017-05-24 10:53:00",
-          description: "'Pair Programming Illuminated' by Laurie Williams and Robert Kessler",
-          category: "Book",
-          amount: 1498
-        },
-        {
-          id: 4,
-          posted_at: "2017-05-24 08:52:00",
-          description: "Medium Iced Cold Brew, Gregory's Coffee",
-          category: "Coffee",
-          amount: 365
-        }
-      ]
+      transactions: []
     }
+    this.fetchTransactions = this.fetchTransactions.bind(this)
+    this.handleChange = this.handleChange.bind(this)
   }
 
   handleChange(event) {
-    // your code here
+    this.setState({
+      searchTerm: event.target.value.toLowerCase()
```

Comment:
may want to store the value of the search term exactly as the user typed it. Otherwise, it will be confusing if they type a capital letter and it shows up as a lower case letter.